rasgotransforms/rasgotransforms/main.py

```diff
@@ -77,31 +82,30 @@ def serve_rasgo_transform_templates() -> List[TransformTemplate]:
             )
     return template_list
 
-def load_all_yaml_files() -> Dict[str, Dict[str, Dict]]:
+def load_all_yaml_files(
+        datawarehouse: str = None
```

Comment:
Same here about having `datawarehouse` here be required param/not None or make a Enum

Reply:
good call on the enum